stories/Grid.story.jsx

```diff
@@ -37,6 +42,38 @@ storiesOf("Grid", module)
       </Row>
     ))
   )
+  .addWithJSX(
+    "responsive",
+    withInfo()(() => (
+      <Row position="center">
+        <Col xs={10} sm={3} md={7} lg={3}>
+          <Box />
+        </Col>
+        <Col xs={10} sm={3} md={8} lg={3}>
+          <BoxBlue />
+        </Col>
+        <Col xs={10} sm={3} md={7} lg={3}>
+          <Box />
+        </Col>
+      </Row>
+    ))
+  )
+  .addWithJSX(
+    "responsive with none",
+    withInfo()(() => (
+      <Row position="center">
+        <Col xs={7} sm={12} md={7} lg={3}>
+          <Box />
+        </Col>
+        <Col xs={7} sm="none" md={10} lg={3}>
```

Comment:
Would rather just leave out sm as a property to denote “none” rather than have “none”

Reply:
Actually if "none" means "to hide" then I think we should think about renaming this. Otherwise I assume it means I don't want anything set for "sm" mode, rather than hide the column when small

Reply:
The default prop is `0`. zero means the column takes the width of the content inside it.
Will add a lot of boilerplate code if we set default to `none`.